cli/README.md

```diff
@@ -224,6 +224,108 @@ model User {
 
 During introspection, the Canonical Schema Mapping is used to decide if a field should have the `@updatedAt` attribute applied. It dictates that this should never happen.
 
+### Introspection Configuration
+
+The Canonical Schema Mapping is designed to ensure that the introspection result is always easy to understand. It is a design goal to not perform any magic. As a result, often it is required to manually extend the schema after introspecting a database. This is tedious and leads to extra work when re-introspecting, because manual edits will be lost.
+
+Introspection Configuration allows the user to change the Canonical Schema Mapping to be in line with patterns used in the database. 
+
+The following configuration options are awailable:
+
+- **fieldPatterns**: Add attributes to a field by matching on its name
+- **modelPatterns**: Add attributes to a model by matching on its name
+
+The configuration is stored in a `prismarc.json` file next to the `schema.prisma` file. The following example will
+
+- add the `@updatedAt` attribute to all fields called updatedAt or updated_at
+- add the `@createdAt` attribute to all fields called createdAt
+- add `@id` and `@default(cuid)` attributes to all fields that end with _id.
+
+```
+{
+  "introspection": {
+    fieldPatterns: {
+      "@updatedAt": "/^updatedAt|updated_at$/",
+      "@createdAt": "/^createdAt$/",
+      "@id @default(cuid())": "/_id$/",
+    }
+  }
+}
```

Comment:
I would like to be able to set how columns name should be mapped when invalid key is encountered.

```
{
  invalidCharactersReplacements: {
    caseInsensitive: {
      "€": ["eur"],
      "a": ["á"],
      "c": ["č"],
      "e": ["é", "ě"],
      "i": ["í"],
      ...
    },
    caseSensitive: { ... }
  },
}
```

Possibly use https://stackoverflow.com/a/18391901/3001856 (It is easier to write, and